src/response/trowtoken.rs

```diff
@@ -0,0 +1,100 @@
+use failure::{Error};
+use std::io::Cursor;
+use std::time::{SystemTime, UNIX_EPOCH};
+use rocket::http::{Status};
+use rocket::http::{ContentType};
+use rocket::request::Request;
+use rocket::response::{Responder, Response};
+use crypto::sha2::Sha256;
+use chrono::Local;
+use jwt::{Header as TokenHeader, Token};
+
+const AUTHORISATION_SECRET: &str = "Bob Marley Rastafaria";
```

Comment:
😂 I worry about you sometimes

Reply:
I put this secret in twice where it needs to decode and encode the token. I need to work out the best way of having it somewhere once in Rust! hmmm I wonder what music I was listening to that day.

Reply:
Yeah, I mention it below as well. I actually know how to solve this one. It's one of those things that's confusing at first, but makes sense later. You can look up state in the guide if you want to know more, but basically we'll just tell rocket to manage some struct with our secrets.

Reply:
I'll close this for the minute and we'll deal with it in a future PR